src/WooCommerceCheckout.php

```diff
@@ -28,6 +28,36 @@ public static function init() {
       add_filter('woocommerce_checkout_fields', __CLASS__ . '::addConfirmationEmailCheckoutField');
       add_action('woocommerce_checkout_process', __CLASS__ . '::checkConfirmationEmailField');
     }
+
+    add_filter( 'woocommerce_form_field', __CLASS__ . '::woocommerceFormField', 10, 4 );
```

Comment:
For the hook we usually add a small comment what it's used for. This you don't have to jump to the function to see if it's the right one. (Some hooks might miss the comments but we should get more consistent with it)

```suggestion

    // Add checkout error messages.
    add_filter( 'woocommerce_form_field', __CLASS__ . '::woocommerceFormField', 10, 4 );
```

Reply:
OK I get that, but in this case you mean put 2 different comments in general? like one short description in the hook and then the whole thing in the function?